ivy/container/utility.py

```diff
@@ -10,44 +10,92 @@
 
 # noinspection PyMissingConstructor
 class ContainerWithUtility(ContainerBase):
+    @staticmethod
+    def static_all(
+        x: Union[ivy.Array, ivy.NativeArray, ivy.Container],
+        axis: Optional[Union[int, Sequence[int], ivy.Container]] = None,
+        keepdims: Union[bool, ivy.Container] = False,
+        key_chains: Optional[Union[Sequence[str], Dict[str, str]]] = None,
+        to_apply: bool = True,
+        prune_unapplied: bool = False,
+        map_sequences: bool = False,
+        *,
+        out: Optional[ivy.Container] = None,
+    ) -> ivy.Container:
+        return ContainerBase.multi_map_in_static_method(
+            "all",
+            x,
+            axis,
+            keepdims,
+            key_chains=key_chains,
+            to_apply=to_apply,
+            prune_unapplied=prune_unapplied,
+            map_sequences=map_sequences,
+            out=out,
+        )
+
     def all(
         self: ivy.Container,
-        axis: Optional[Union[int, Tuple[int], List[int]]] = None,
-        keepdims: bool = False,
-        key_chains: Optional[Union[List[str], Dict[str, str]]] = None,
+        axis: Optional[Union[int, Sequence[int], ivy.Container]] = None,
+        keepdims: Union[bool, ivy.Container] = False,
+        key_chains: Optional[Union[Sequence[str], Dict[str, str]]] = None,
+        to_apply: bool = True,
+        prune_unapplied: bool = False,
+        map_sequences: bool = False,
+        out: Optional[ivy.Container] = None,
```

Comment:
should have `*` above `out` here, because `out` should always be keyword-only, even for instance methods.